src/views/course-edit-tools/course-feedback.vue

```diff
@@ -0,0 +1,139 @@
+<!--
+Filename: course-feedback.vue
+Use: download received feedback
+Author: nv
+Date: September 08, 2022
+Dependencies:
+  vuex,
+  @/mixins/locale.vue
+-->
+
+<template>
+  <div class="row mt-5">
+    <div class="col">
+      <b-button
+        size="sm"
+        variant="success"
+        :class="langIsAr? 'float-left' : 'float-right'"
+        @click="$bvModal.show('download-feedback')"
+      >
+        <i class="fas fa-info-circle"></i> {{ y18n('feedback.downloadFeedback') }}
+      </b-button>
+    </div>
+    <div class="col text-dark">
+      {{ y18n('feedback.feedbackTip') }}
+    </div>
+    <b-modal
+      id="download-feedback"
+      :title="y18n('feedback.modal.title')"
+      header-bg-variant="warning"
+      ok-variant="warning"
+      :ok-title="y18n('feedback.modal.ok')"
+      :cancel-title="y18n('cancel')"
+      centered
+      :aria-label="y18n('popupwarning')"
+      @ok="printPDF"
+    >
+    </b-modal>
+  </div>
+</template>
+
+<script>
+
+import { mapActions, mapGetters } from 'vuex'
+import { storeHandler, locale } from '@/mixins'
+import { jsPDF } from 'jspdf'
+import http from 'axios'
+
+export default {
+  name: 'CourseFeedback',
+
+  mixins: [
+    storeHandler,
+    locale
+  ],
+
+  data () {
+  },
+
+  computed: {
+    ...mapActions([
+      'fetchEnrollment'
+    ]),
+    ...mapGetters([
+      'enrollmentFeedback',
+      'courseId'
+    ])
+  },
+
+  methods: {
+    /**
+     * Function printPDF: prints feedback in a PDF tp download
+     * Author: nv
+     * Last Updated: September 14, 2022
+     */
+    async printPDF () {
+      if (typeof this.enrollmentFeedback !== 'undefined') {
+        /* eslint-disable */
+        const doc = new jsPDF('p','pt')
+        /* eslint-enable */
+        const id = this.courseId
+        let name
+        let feedback = []
+        const x = 30
+        let y = 30
+        const lineheight = doc.getTextDimensions('Sample Text').h + 5
+        let blockheight = 0
+
+        await http.get(`courses/getCourseName?courseId=${id}`)
+          .then(({ data }) => {
+            console.log(data.courseName)
+            name = data.courseName
+          })
+        await http.get(`enrollments/getAllByCourseId?courseId=${id}`)
+          .then(({ data }) => {
+            console.log(data.subs[0].feedback)
+            feedback = data.subs[0].feedback
+          })
```

Comment:
this should be moved to a store action

it furthermore just takes the feedback from first subscriber